e2e/cjs/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "schedule-test-cjs",
```

Comment:
Seems we have a split between "normal JavaScript package" and "ESM style package"? I don't really understand this yet.

Reply:
yes, the package is now compatible to be used in esm javascript module sytem, which is the new standard way or javascript for node js.

A lot packages are starting to be esm only, but renovate is not ready yet, so we publish in both types.

When renovate is ready for esm only, we can drop the commonjs version

Reply:
Thank you for the explanation! 😄